recipes/nolearn/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nolearn-{{ version }}.tar.gz
+  sha256: bab2dc2aeb9560264791298f1d206b78fd9b345cecdd500b34f469d367765158
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python ==2.7.*
```

It looks like the upstream package supports Python 3 just fine. We generally aren't building for Python 2 any more. Is there any reason you're locking this to Python 2 rather than say:
```suggestion
    - python >=3.6
```